core/capabilities/remote/types/config.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+const (
+	DefaultRegistrationRefreshMs = 30_000
+	DefaultRegistrationExpiryMs  = 120_000
+	DefaultMessageExpiryMs       = 120_000
+)
+
+// NOTE: consider splitting this config into values stored in Registry (KS-118)
+// and values defined locally by Capability owners.
```

Comment:
+1. This should be capability specific configuration. Or not configurable at all.

Reply:
Yeah, we can move to Registry once that's ready.